addons/account_check_printing/views/account_journal_views.xml

```diff
@@ -36,4 +36,17 @@
                 </xpath>
             </field>
         </record>
+        <record id="view_account_bank_journal_form_inherited" model="ir.ui.view">
```

Comment:
Why you need this view? There is already a view for **account.journal**. See this: https://github.com/odoo/odoo/pull/15436/files#diff-d87c26a4a6f91e4d37f7ec1072df44b3R24 .